site/index.html

```
@@ -393,14 +407,14 @@ <h2>🎬 功能展示</h2>
     </div>
 
     <!-- Lightbox 彈出層 -->
-    <div id="lightbox" class="lightbox" onclick="closeLightbox(event)">
-      <span class="lightbox-close" onclick="closeLightbox(event)">&times;</span>
-      <span class="lightbox-nav lightbox-prev" onclick="navigateLightbox(event, -1)">&#10094;</span>
+    <div id="lightbox" class="lightbox" role="dialog" aria-modal="true" aria-hidden="true">
+      <span class="lightbox-close" aria-label="關閉大圖">&times;</span>
+      <span class="lightbox-nav lightbox-prev" aria-label="上一張">&#10094;</span>
       <div class="lightbox-content">
         <img id="lightbox-img" src="" alt="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)